src/commands/index.ts

```diff
@@ -125,8 +125,10 @@ import { deleteWarn } from './moderation/warn/deleteWarn';
 import { atc } from './a32nx/atc';
 import { market } from './support/market';
 import { takeoffIssues } from './a32nx/takeoffissues';
+import { dfd } from './general/dfd';
 import { simbridge } from './support/simbridge';
 
+
```

Comment:
sorry I messed up when fixing the conflicts, 
could you please remove this line I added